app/controllers/web/news_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Web::NewsController < Web::ApplicationController
+  def index
+    @news = NewsDecorator.decorate_collection News.published.order('published_at DESC')
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.